.github/workflows/build-docker-full.yml

```diff
@@ -16,15 +16,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
```

Comment:
should we start pinning these to specific commits like in other actions? or are we ok b/c these are maintained by GH?

Reply:
That was my feeling on these. Since they are maintained by GH I think they are trustworthy